api/app/Models/Contact.ts

```diff
@@ -0,0 +1,77 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column, beforeCreate } from '@ioc:Adonis/Lucid/Orm'
+import {cuid} from '@ioc:Adonis/Core/Helpers'
+import { attachment, AttachmentContract } from '@ioc:Adonis/Addons/AttachmentLite'
+
+export default class Contact extends BaseModel {
+
+  public static selfAssignPrimaryKey = true
+
+  @column({isPrimary: true})
+  public id: string
+
+  @column({serializeAs: 'firstName'})
+  public firstName: string
+
+  @column()
+  public surname: string
+
+  @column()
+  public company?: string | null | undefined
```

Comment:
This applies to all column properties in this class.

- [ ] Remove the optional operator (?) from the properties.
- [ ] Remove `undefined` type.